shelley/chain-and-ledger/shelley-spec-ledger-test/tmp92599-9

```diff
@@ -0,0 +1,293 @@
+output = relay
```

Comment:
Included by accident?

Reply:
Looks like.  Should there be a `.gitignore` entry for these files?

Reply:
I've never seen this before, so I'm not sure where it's coming from

Reply:
I've removed it.  Thanks!